cypress/integration/teacher-training-adviser.spec.js

```diff
@@ -97,6 +98,79 @@ describe("Feature - Get an adviser : Tests execution date and time : " + new Dat
 		});
 	});
 
+	it('It shows "Get support" message to the UK returner if he selects subject as "Other"', function () {
+		cy.enterFirstNameLastNameandEmail(
+			this.testData.firstName,
+			this.testData.lastName,
+			this.testData.email
+		);
+		cy.returningToTeaching(true);
+		cy.havePreviousTeacherReferenceNumber(true);
+		cy.enterPreviousTeacherReferenceNumber(23478463);
+		cy.selectPreviuosMainSubject("Computing");
+		cy.selectSubjectLikeToTeach("Other");
+		cy.get(".govuk-heading-l").should("exist").should("have.text", "Get support");
+	});
+
+	it('It navigates to mailinglist sign up page if user clicks on "send you information via email " link', function () {
+		cy.enterFirstNameLastNameandEmail(
+			this.testData.firstName,
+			this.testData.lastName,
+			this.testData.email
+		);
+		cy.returningToTeaching(true);
+		cy.havePreviousTeacherReferenceNumber(true);
+		cy.enterPreviousTeacherReferenceNumber(23478463);
+		cy.selectPreviuosMainSubject("Computing");
+		cy.selectSubjectLikeToTeach("Other");
+		cy.get(".govuk-heading-l").should("exist").should("have.text", "Get support");
+		cy.contains("a", "send you information via email").click();
+		cy.url().then((url) => {
+			expect(url).contains(Navlinks.mailingListSignupUrl);
+		});
+		cy.get("h1")
+			.should("exist")
+			.should("have.text", "Sign up for email updates to help you get into teaching");
+	});
+	it('It navigates to mailinglist sign up page if user clicks on "attending a return to teaching event" link', function () {
```

Comment:
This is a bug I'm in the process of fixing

Reply:
ohh, I'll update this accordingly.

Reply:
For the time being I'm skipping this test , will implement once bug is resolved.